### PR TITLE
Website MixMods added

### PR DESCRIPTION
**Pull request by PeeQueno**
A Presence for site www.mixmods.com.br, the biggest site of mods from America Latina
image

### CODE REVIEW

File: websites/M/MixMods/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "service": "MixMods",
+    "description": {
+      "en": "MixMods is a website of mods for GTA San Andreas and another games. Created by Junior_Djjr, is the biggest website of mods of America Latina",
+      "pt": "MixMods é um site de mods para GTA San Andreas e outros jogos. Criado por Junior_Djjr, é o maior site de mods da America Latina."
+    }, 
+    "url": "www.mixmods.com.br",
+    "regExp": "www.mixmods.com.br",
+    "version": "1.0.0",
+    "logo": "https://3.bp.blogspot.com/-gel6b0W1JF0/XA5SaECGfmI/AAAAAAAAUJY/bm9_20Dktkc-f23uP1WjsInwUUVnDgkgwCK4BGAYYCw/s1600/mixmods-logo-min.png",
+    "thumbnail": "https://3.bp.blogspot.com/-gel6b0W1JF0/XA5SaECGfmI/AAAAAAAAUJY/bm9_20Dktkc-f23uP1WjsInwUUVnDgkgwCK4BGAYYCw/s1600/mixmods-logo-min.png",
```
Comment (frozaum) [inline websites/M/MixMods/dist/metadata.json:15]: Please host the images somewhere else (like Imgur or some file hosting provider) and change the thumbnail to a promotional or a screenshot image.

File: websites/M/MixMods/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "name": "Pequeno",
+      "id": "323577179814494209"
+    },
+    "service": "MixMods",
+    "description": {
+      "en": "MixMods is a website of mods for GTA San Andreas and another games. Created by Junior_Djjr, is the biggest website of mods of America Latina",
+      "pt": "MixMods é um site de mods para GTA San Andreas e outros jogos. Criado por Junior_Djjr, é o maior site de mods da America Latina."
+    }, 
+    "url": "www.mixmods.com.br",
+    "regExp": "www.mixmods.com.br",
```
Comment (Alanexei) [inline websites/M/MixMods/dist/metadata.json:12]: Remove regExp

**Comment by PeeQueno** [comment]
The changes request has been placed

File: Sites/M/MixMods/presence.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      presenceData.state = name;
+  } else { 
+      presenceData.details = ("Navegando no site");
+      const url = document.location.href.split("#")[1];
+      const text = url.split("=")[1];
+      presenceData.state = `Página ${text}`;
+  }
+
+  presence.setActivity(presenceData);
+});
```
Comment (Bas950) [inline Sites/M/MixMods/presence.ts:43]: Your whole ts file is not in the correct folder.

**Comment by PeeQueno** [comment]
Ok, sorry, fixed the folder directory...

File: websites/M/MixMods/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "service": "MixMods",
+    "description": {
+      "en": "MixMods is a website of mods for GTA San Andreas and another games. Created by Junior_Djjr, is the biggest website of mods of America Latina",
+      "pt": "MixMods é um site de mods para GTA San Andreas e outros jogos. Criado por Junior_Djjr, é o maior site de mods da America Latina."
+    }, 
+    "url": "www.mixmods.com.br",
+    "version": "1.0.0",
+    "logo": "https://imgur.com/7YvTrtD",
+    "thumbnail": "https://imgur.com/x17ShNm",
```
Comment (frozaum) [inline websites/M/MixMods/dist/metadata.json:14]: Please use the **DIRECT** link of the image.

Comment (PeeQueno) [reply websites/M/MixMods/dist/metadata.json]: Done

**Pull request merged**